src/wired/component.tsx

```diff
@@ -175,6 +183,11 @@ async function getServerInitialProps<Props extends WiredProps, ServerSideProps>(
 
   await ensureQueryFlushed(preloadedQuery);
 
+  if (serverSidePostQuery) {
+    const queryResult = await preloadedQuery.source?.toPromise();
```

Comment:
Will this cause a double network request? I'm not sure if the observables returned from Relay are hot or cold.

Reply:
No, call to `preloadedQuery.source?.toPromise()` here results in a already resolved promise.